### PR TITLE
WIP: Updatereleasenotes

### PR DESCRIPTION
**Pull request by Mac-Rae**
Adds the WIP Release notes for SuiteCRM Release

### CODE REVIEW

**Comment by connorshea** [comment]
Could we include the warning about dropping PHP 5.5 here? 🙏

File: content/admin/releases/7.10.x/_index.en.adoc [hunk 1]
```diff
@@ -9,6 +9,205 @@ weight: 9890
...
+* PR: https://github.com/salesagility/SuiteCRM/pull/7525[7525^] - API Create Relationship via Link
+* PR: https://github.com/salesagility/SuiteCRM/pull/7515[7515^] - Scheduled Reports: Fix report name relation and popup search
+* PR: https://github.com/salesagility/SuiteCRM/pull/7428[7428^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7427[7427^] - Show logs lines that was made by anonymous
+* PR: https://github.com/salesagility/SuiteCRM/pull/7195[7195^] - Inspections compatibility
+* PR: https://github.com/salesagility/SuiteCRM/pull/7193[7193^] - Remove Unused Import
+* PR: https://github.com/salesagility/SuiteCRM/pull/7141[7141^] - Type casting
+* PR: https://github.com/salesagility/SuiteCRM/pull/6765[6765^] - Issue: https://github.com/salesagility/SuiteCRM/issues/321[321^] - Hitting enter in the password input saves the user but not the password
+* PR: https://github.com/salesagility/SuiteCRM/pull/6750[6750^] - Issue: https://github.com/salesagility/SuiteCRM/issues/4754[4754^] - Remove PHP4 style constructors
+* PR: https://github.com/salesagility/SuiteCRM/pull/6503[6503^] - Add a SAML2 metadata endpoint
```
Comment (connorshea) [inline content/admin/releases/7.10.x/_index.en.adoc]: This should probably be an enhancement?

File: content/admin/releases/7.10.x/_index.en.adoc [hunk 2]
```diff
@@ -9,6 +9,205 @@ weight: 9890
...
+* PR: https://github.com/salesagility/SuiteCRM/pull/7802[7802^] - Issue: https://github.com/salesagility/SuiteCRM/issues/6830[6830^] - Code coverage as a separate stage in CI
+* PR: https://github.com/salesagility/SuiteCRM/pull/7797[7797^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7779[7779^] - PHP Fatal error in modules/Connectors
+* PR: https://github.com/salesagility/SuiteCRM/pull/7783[7783^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7780[7780^] - Bad css format in Date and Date Range Inputs in search forms
+* PR: https://github.com/salesagility/SuiteCRM/pull/7782[7782^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7781[7781^] - Now we can compile SuiteP only one color_scheme
+* PR: https://github.com/salesagility/SuiteCRM/pull/7777[7777^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7784[7784^] - Grouping by with xxx_usdollar currency fields
+* PR: https://github.com/salesagility/SuiteCRM/pull/7774[7774^] - EmailMarketing: Add security groups support
```
Comment (connorshea) [inline content/admin/releases/7.10.x/_index.en.adoc]: Enhancement?

File: content/admin/releases/7.10.x/_index.en.adoc [hunk 3]
```diff
@@ -9,6 +9,205 @@ weight: 9890
...
+* PR: https://github.com/salesagility/SuiteCRM/pull/7929[7929^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7928[7928^] - Upgrade wizard recommends composer update instead of composer install
+* PR: https://github.com/salesagility/SuiteCRM/pull/7925[7925^] - Enable Delete button in Actions menu
+* PR: https://github.com/salesagility/SuiteCRM/pull/7924[7924^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7923[7923^] - Verify the variable is an array
+* PR: https://github.com/salesagility/SuiteCRM/pull/7922[7922^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7880[7880^] - InboundEmail mime parser
+* PR: https://github.com/salesagility/SuiteCRM/pull/7918[7918^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7917[7917^] - Issue with french translation
+* PR: https://github.com/salesagility/SuiteCRM/pull/7913[7913^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7912[7912^] - Avoid PHP Notices in getVardefs() method
+* PR: https://github.com/salesagility/SuiteCRM/pull/7910[7910^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7885[7885^] - Add a SECURITY.md to the repository
```
Comment (connorshea) [inline content/admin/releases/7.10.x/_index.en.adoc]: Enhancement?

File: content/admin/releases/7.10.x/_index.en.adoc [hunk 4]
```diff
@@ -9,6 +9,205 @@ weight: 9890
...
+* PR: https://github.com/salesagility/SuiteCRM/pull/8061[8061^] - Issue: https://github.com/salesagility/SuiteCRM/issues/6314[6314^] - Unused language strings in ver. 7.10.8
+* PR: https://github.com/salesagility/SuiteCRM/pull/8060[8060^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7987[7987^] - Apache log
+* PR: https://github.com/salesagility/SuiteCRM/pull/8059[8059^] - Added a check for SUGARCRM restrictions in htaccess
+* PR: https://github.com/salesagility/SuiteCRM/pull/8058[8058^] - Issue: https://github.com/salesagility/SuiteCRM/issues/8057[8057^] - Deprecated usages of implode
+* PR: https://github.com/salesagility/SuiteCRM/pull/8056[8056^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7128[7128^] - Remove scheme to avoid mixed content error
+* PR: https://github.com/salesagility/SuiteCRM/pull/8054[8054^] - Improve footer styling for new stats item
+* PR: https://github.com/salesagility/SuiteCRM/pull/8051[8051^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7397[7397^] - Implement Refresh Token Grant
```
Comment (connorshea) [inline content/admin/releases/7.10.x/_index.en.adoc]: Enhancement?

File: content/admin/releases/7.10.x/_index.en.adoc [hunk 5]
```diff
@@ -9,6 +9,205 @@ weight: 9890
...
+* PR: https://github.com/salesagility/SuiteCRM/pull/7529[7529^] - Codacy
+* PR: https://github.com/salesagility/SuiteCRM/pull/7525[7525^] - API Create Relationship via Link
+* PR: https://github.com/salesagility/SuiteCRM/pull/7515[7515^] - Scheduled Reports: Fix report name relation and popup search
+* PR: https://github.com/salesagility/SuiteCRM/pull/7428[7428^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7427[7427^] - Show logs lines that was made by anonymous
+* PR: https://github.com/salesagility/SuiteCRM/pull/7195[7195^] - Inspections compatibility
+* PR: https://github.com/salesagility/SuiteCRM/pull/7193[7193^] - Remove Unused Import
+* PR: https://github.com/salesagility/SuiteCRM/pull/7141[7141^] - Type casting
+* PR: https://github.com/salesagility/SuiteCRM/pull/6765[6765^] - Issue: https://github.com/salesagility/SuiteCRM/issues/321[321^] - Hitting enter in the password input saves the user but not the password
+* PR: https://github.com/salesagility/SuiteCRM/pull/6750[6750^] - Issue: https://github.com/salesagility/SuiteCRM/issues/4754[4754^] - Remove PHP4 style constructors
```
Comment (connorshea) [inline content/admin/releases/7.10.x/_index.en.adoc]: This should be in the Development section.

File: content/admin/releases/7.10.x/_index.en.adoc (hunk 1 above)
Comment (Mac-Rae) [reply content/admin/releases/7.10.x/_index.en.adoc]: I'm basing them off the tags they were given with so I could be incorrect however the once above I believe are things that should have been there in the first place/ functionality that didn't work hence the Bug Fixes section

File: content/admin/releases/7.10.x/_index.en.adoc (hunk 3 above)
Comment (Mac-Rae) [reply content/admin/releases/7.10.x/_index.en.adoc]: Sorted a few of these

File: content/admin/releases/7.10.x/_index.en.adoc (hunk 4 above)
Comment (Mac-Rae) [reply content/admin/releases/7.10.x/_index.en.adoc]: Im not sure any of these are?

File: content/admin/releases/7.10.x/_index.en.adoc [hunk 6]
```diff
@@ -9,6 +9,205 @@ weight: 9890
...
+* PR: https://github.com/salesagility/SuiteCRM/pull/8031[8031^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7758[7758^] - Disable Action menu has no effect on menus in subpanel
+* PR: https://github.com/salesagility/SuiteCRM/pull/8030[8030^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7738[7738^] - Email Template selection in email module is not working in Edge/IE11
+* PR: https://github.com/salesagility/SuiteCRM/pull/8029[8029^] - Updated mkdir calls to throw RuntimeExceptions
+* PR: https://github.com/salesagility/SuiteCRM/pull/8028[8028^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7874[7874^] - Unable to use custom _head.tpl file
+* PR: https://github.com/salesagility/SuiteCRM/pull/8027[8027^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7882[7882^] - No 'Server response time' in SuiteP
+* PR: https://github.com/salesagility/SuiteCRM/pull/8026[8026^] - Issue: https://github.com/salesagility/SuiteCRM/issues/8025[8025^] - OAuth Keys Fixed a grammatical error in include/templates/Template.php OAuth2 Clients and Tokens icons are missing
```
[the remark this answers is not in the document]
Comment (Mac-Rae) [reply content/admin/releases/7.10.x/_index.en.adoc]: Resolved

File: content/admin/releases/7.10.x/_index.en.adoc (hunk 5 above)
Comment (Mac-Rae) [reply content/admin/releases/7.10.x/_index.en.adoc]: Sorted

File: content/admin/releases/7.10.x/_index.en.adoc (hunk 2 above)
Comment (Mac-Rae) [reply content/admin/releases/7.10.x/_index.en.adoc]: Sorted

File: content/admin/releases/7.11.x/_index.en.adoc [hunk 7]
```diff
@@ -9,6 +9,207 @@ weight: 9880
 
 :experimental:
 
+== 7.10.21
+
+_Released 01/11/2019_
```
Comment (likhobory) [inline content/admin/releases/7.11.x/_index.en.adoc]: It should be
```
== 7.11.9

_Released 04/11/2019_
```

File: content/admin/releases/7.11.x/_index.en.adoc [hunk 8]
```diff
@@ -9,6 +9,207 @@ weight: 9880
...
+
+=== pass:[<i class="fa fa-files-o fa-1x"></i>] Assets
+
+* https://github.com/salesagility/SuiteCRM/archive/v7.10.21.zip[*Source code* (zip)]
+* https://github.com/salesagility/SuiteCRM/archive/v7.10.21.tar.gz[*Source code* (tar.gz)]
```
Comment (likhobory) [inline content/admin/releases/7.11.x/_index.en.adoc]: Invalid URLs to v7.10.21

**Pull request merged**